cmd/agent/gui/checks.go

```diff
@@ -185,56 +199,49 @@ func setCheckConfigFile(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	// Write new configs to custom checks directory
+	// Check if this file is in a subdir (file name is of the form `{check name}: {file name}`)
```

Comment:
This is duplicate from `getCheckConfigFile` we could make it a function on it's own